mlflow/utils/autologging_utils/__init__.py

```diff
@@ -296,23 +295,6 @@ def batch_metrics_logger(run_id):
     batch_metrics_logger.flush()
 
 
-def gen_autologging_package_version_requirements_doc(integration_name):
```

Comment:
I think this part is important to be added in autolog doc. It shows the supported version range for autolog, it shouldn't be removed.

Reply:
I understand.  The issue I'm having is certainly an edge case.

Reply:
We try best effort to append the autolog doc with supported version range , if fail we can skip appending it.